test/testdata/fixtures/multi-dc-reaper/k8ssandra.yaml

```diff
@@ -16,9 +26,11 @@ spec:
   reaper:
     keyspace: reaper_ks # custom name
     cassandraUserSecretRef:
-      name: reaper-cql-secret # pre-existing secret
+      name: test-cql-secret # pre-existing secret
```

Comment:
Why this change? The name can be anything.

Reply:
Oh I see, you hardcoded the secret name in `getReaperUiSecret`. I think you should remove that function completely. The test code knows what the secret name is, so you just create a `ClusterKey` object with the right name and call `getSecret`.